Resources/doc/index.md

```diff
@@ -180,6 +194,7 @@ The following documents are available:
 - [Functionally testing a JWT protected api](3-functional-testing.md)
 - [Working with CORS requests](4-cors-requests.md)
 - [JWT encoder service customization](5-encoder-service.md)
-- [Extending JWTTokenAuthenticator](6-extending-jwt-authenticator.md)
+- [Extending Authenticator](6-extending-jwt-authenticator.md)
 - [Creating JWT tokens programmatically](7-manual-token-creation.md)
 - [A database-less user provider](8-jwt-user-provider.md)
+- [Decode JWT token](9-decode-jwt-token.md)
```

Comment:
I fear this might be slightly confusing. What about `Accessing the authenticated JWT token`?

Reply:
Great, i will change.